java-sdk/pom.xml

```diff
@@ -0,0 +1,157 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.sagebionetworks.bridge</groupId>
+        <artifactId>sdk-group</artifactId>
+        <version>0.7-SNAPSHOT</version>
```

Comment:
Add comment about mvn versions:set -DnewVersion=0.7.0 to set version in all sub-projects and parent pom.xml